commons-db/src/main/java/com/palantir/nexus/db/sql/SqlConnectionHelper.java

```diff
@@ -212,15 +214,21 @@ Long selectLongWithDefault(Connection c, String key, Long defaultVal, Object...
 
     AgnosticLightResultSet selectLightResultSetUnregisteredQuery(Connection c,
                                                                  String sql,
-                                                                 Object... vs)
-            throws PalantirSqlException, PalantirInterruptedException {
+                                                                 Object... vs) {
+        return selectLightResultSetUnregisteredQueryWithFetchSize(c, sql, null, vs);
+    }
+
+    AgnosticLightResultSet selectLightResultSetUnregisteredQueryWithFetchSize(Connection c,
+                                                                              String sql,
+                                                                              @Nullable Integer fetchSize,
+                                                                              Object... vs) {
         return basicSql.selectLightResultSetSpecifyingDBType(
                 c,
                 SQLString.getUnregisteredQuery(sql),
                 vs,
-                DBType.getTypeFromConnection(c));
+                DBType.getTypeFromConnection(c),
+                fetchSize);
     }
-
```

Comment:
Yeah sorry about that :(. Will probably not open a new PR to insert one blank line